src/app/app.component.ts

```diff
@@ -26,4 +59,10 @@ export class AppComponent implements OnInit {
       }
     });
   }
+
```

Comment:
You have all logic in 1 method ngOnInit, please separate all to extra methods and put method in ngOnInit

Reply:
fixed